### ISSUE TITLE
Pin spacing guide missing

### ISSUE THREAD

**Issue by dp138504** [deviantollam/LishiKeyCuttingGuides#1]
Firstly, thank you for posting this, it's great!

I was wondering if you could also include the pin spacing guide you used at [3:22](https://youtu.be/MVNUbwL26H4?t=202) in the video you posted about this.

Would you also be willing to upload the case as well?

Thanks again for this!

**Comment by deviantollam** [comment]
Ah, I can see about finding that file.  For now, it's not entirely  necessary, since the notion is for any keys used in such a system to be  pre-made with 0-0-0-0-0 bitting (there is no real way for the cutters to  make 0-cuts) and thus a person can, in theory, drop marker lines  directly down on the blade from those pre-scored cut positions.

We'll probably do the same with Kwikset when we release that... even  though there will likely be a "1" sized guide, it will be expected that  most folk are starting with keys cut to 1-1-1-1-1 already.

**Comment by dp138504** [comment]
I'll have to give it a shot with an all-zero bitting as you suggested. I appreciate you trying to find that file nonetheless!

**Comment by xcel102** [comment]
I would also like to have the spacing guide. Thank you for making these!

**Comment by Misfitsman805** [comment]
Spacing guide would make this complete! Thanks for this deviantollam

**Comment by cgkades** [comment]
Came here to make the same request, and possibly add a request for the case too? 

For anyone interested there are some good finds on thiniverse if you search Lishi. I did print the KW1, and it seemed to work ok. I had to file the key I made from the KW1 a little though.

examples:
[KW1](https://www.thingiverse.com/thing:5163333)
[SC1](https://www.thingiverse.com/thing:5164225)

**Comment by TheMitch2** [comment]
> Ah, I can see about finding that file. For now, it's not entirely necessary, since the notion is for any keys used in such a system to be pre-made with 0-0-0-0-0 bitting (there is no real way for the cutters to make 0-cuts) and thus a person can, in theory, drop marker lines directly down on the blade from those pre-scored cut positions. We'll probably do the same with Kwikset when we release that... even though there will likely be a "1" sized guide, it will be expected that most folk are starting with keys cut to 1-1-1-1-1 already.

Any luck in finding the file yet?

**Comment by PLA-Chris** [comment]
> Came here to make the same request, and possibly add a request for the case too?
> 
> For anyone interested there are some good finds on thiniverse if you search Lishi. I did print the KW1, and it seemed to work ok. I had to file the key I made from the KW1 a little though.
> 
> examples: [KW1](https://www.thingiverse.com/thing:5163333) [SC1](https://www.thingiverse.com/thing:5164225)

Yeah I printed one of these of for marking keys as even the owner says they are not quite ready for real world use.

**Comment by zrieprakis** [comment]
Same req as everyone else, but completely understand if the sharpie guide is just available in the paid package.  Seems that the package has been sold out for some time, though, so there may be other issues here.

**Comment by ThisGuyNeedsABeer** [comment]
> > Ah, I can see about finding that file. For now, it's not entirely necessary, since the notion is for any keys used in such a system to be pre-made with 0-0-0-0-0 bitting (there is no real way for the cutters to make 0-cuts) and thus a person can, in theory, drop marker lines directly down on the blade from those pre-scored cut positions. We'll probably do the same with Kwikset when we release that... even though there will likely be a "1" sized guide, it will be expected that most folk are starting with keys cut to 1-1-1-1-1 already.
> 
> Any luck in finding the file yet?

Pin spacing guide was uploaded as of last week. I had these printed by Craftcloud, and haven't been able to get a functional SC1 key cut yet with a Kryptonite padlock, or a basic door handle. I'm going to try to get a new set printed, and see what happens now with the pin guide. I was previously using these guides (https://www.ebay.com/itm/325452889334), which out of 20 attempts, produced only 1 working key.

**Comment by ThisGuyNeedsABeer** [comment]
Getting a second round cut, the first one I did with PLA, not sure if it's my Lishi nippers that aren't accurate or the print. Here are the specs for the second attempt:

Material: SLS Nylon PA12
Finish: Polished
Color: Dyed Orange
Cost: $3.24/stl file + $5.00 shipping
Printer: JawsTec found on CraftCloud

Edit: If anyone is following, the print came out flawlessly, the fit is perfect, and the cut worked the first time. 10/10!
